test/parallel/test-fs-close-errors.js

```diff
@@ -17,3 +17,16 @@ const fs = require('fs');
   assert.throws(() => fs.close(input), errObj);
   assert.throws(() => fs.closeSync(input), errObj);
 });
+
+{
+  // Test error when cb is not a function
+  const fd = fs.openSync(__filename, 'r');
+
+  ['', false, null, {}, []].forEach((input) => {
+    const errObj = {
+      code: 'ERR_INVALID_CALLBACK',
+      name: 'TypeError'
+    };
```

Comment:
Minor nit-pick: You can move the `errObj` definition outside of the `forEach()` to below the `fd` definition. This way, it's only being defined once.